source/common/http/http1/codec_impl.cc

```diff
@@ -451,11 +481,17 @@ size_t ConnectionImpl::dispatchSlice(const char* slice, size_t len) {
 }
 
 void ConnectionImpl::onHeaderField(const char* data, size_t length) {
+  ENVOY_CONN_LOG(trace, "onHeaderField: data={}", connection_, absl::string_view(data, length));
```

Comment:
I'm mildly inclined to not log all header fields and values as we log at end of header parsing.  Do you think you'd use this going forward?